persist/src/main/resources/db/migration/V4__NeighborhoodAdds.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO neighborhood (id, name, sqmiles) VALUES
```

Comment:
Since there is already a V4 persist file, you will have to name your new ones V5__Neighborhoodtable and V__5_1 for the adds SQL statement 
That is what is causing the build checks to fail: 
```
Failed to execute goal org.flywaydb:flyway-maven-plugin:6.0.3:migrate (flyway database migration) on project persist: org.flywaydb.core.api.FlywayException: Found more than one migration with version 4
```

Reply:
Thank you! I could not figure out why this was failing or the naming convention for these files.